### PR TITLE
Fraud & Risk tools usability improvements

### LINKED ISSUE #6157: Perform usability updates on Fraud & Risk tools settings
Perform the usability updates described on paJDYF-8mt-p2.

### Additional resources
- WIM5w2Y10SCNBFdMLSVftj-fi-3708_277941

### PR DESCRIPTION
**Pull request by eduardoumpierre**
Fixes #6157

#### Changes proposed in this Pull Request

Perform the usability improvements described on paJDYF-8mt-p2 that are:
- Update the review/block interface to use radio buttons
- Update the title for the metabox on a flagged and approved order
  - Remove callout link 
- Update the advanced settings CTA
  - It should be "Configure" If it's not set yet
  - It should be "Edit" when it's already set

#### Testing instructions

* The unit tests should be passing.
* Checkout to this branch
* Listen to Stripe webhooks `npm run listen`
* Start the client build `npm run start`
* Make sure the fraud settings feature flag is enabled by running `npm run wp option add wcpay_fraud_protection_settings_active 1`
* Go to **Payments > Settings**, scroll to the Fraud protection section
* Select the "Basic" protection level and save the settings
* Refresh the page
* Scroll to the Fraud protection section
* Select the "Advanced" protection level, and click on "Configure"
* Scroll to the "Order items threshold" rule, enable the screening, and set the maximum items to 3
* Set the "Filter action" to "Authorize and hold for review"
* Save the changes
* Click on the "Back to Payment Settings" button
* Scroll to the Fraud protection section
* The "Advanced" protection level CTA button should have the text "Edit" instead of "Configure"
* Go to your test site, add 4 items to the cart, and perform the checkout
* Go to **Payments > Transactions** and click on the "Risk review" tab
* Click on the order you've just created
* Click on "Approve transaction"
* Click on the order id link
* The fraud metabox should be the same as the following

image

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `6221` or branch name `update/6157-usability-improvements` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&branches.woocommerce-payments=update/6157-usability-improvements) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: a2d811befa23306e63b917aa75beb5fad65c6155
- Build time: 2023-05-11 09:10:04 UTC

_Note: the build is updated when a new commit is pushed to this PR._

File: changelog/update-6157-usability-improvements [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: update
+
```
Comment (dwainm) [inline changelog/update-6157-usability-improvements:3]: @eduardoumpierre we probably need to make sure that this changelog is changed so its not included.

File: woocommerce-payments.php [hunk 2]
```diff
@@ -12,7 +12,7 @@
  * WC tested up to: 7.7.0
  * Requires at least: 6.0
  * Requires PHP: 7.3
- * Version: 5.8.1
+ * Version: 5.9.0
```
Comment (dwainm) [inline woocommerce-payments.php]: We probably don't want to bump the version number as part of this PR.

Comment (eduardoumpierre) [reply woocommerce-payments.php]: Fixed on [a6ad876](https://github.com/Automattic/woocommerce-payments/pull/6221/commits/a6ad876d640c78682006402f87f7d4412d3674d2). Thanks for catching this!

File: changelog/update-6157-usability-improvements (hunk 1 above)
Comment (eduardoumpierre) [reply changelog/update-6157-usability-improvements:3]: I think that it won't be included in the final changelog as long as it is a `patch`. Thanks for the comment!

**Review by dwainm — APPROVED** [review]
Works and Code LGTM.

**Pull request merged**